src/NuGet.Core/NuGet.PackageManagement/NuGetPackageManager.cs

```diff
@@ -1761,6 +1772,21 @@ await PreviewBuildIntegratedProjectActionsAsync(buildIntegratedProject, actions,
                                 // use the version exactly as specified in the nuspec file
                                 var packageIdentity = downloadPackageResult.PackageReader.GetIdentity();
 
+                                // Check to see if the hack to install make net461/net462/net47 compatibile with netstandard20 or greater
```

Comment:
nit: Check to see if the hack to make net461/net462/net47 compatibile with netstandard20 or greater